client/jira.go

```diff
@@ -56,6 +56,20 @@ type IssueTypeRequest struct {
 	Name string `json:"name"`
 }
 
+type SearchRequestType struct{
+	Query 	   string 	`json:"jql"`
+	StartIndex int 		`json:"startAt"`
+	MaxResults int 		`json:"maxResults"`
+	Fields	   []string `json:"fields"`
+}
+
+type SearchResult struct{
+	StartIndex   int 			`json:"startAt"`
+	MaxResults 	 int 			`json:"maxResults"`
+	TotalResults int 			`json:"total"`
```

Comment:
Could you run go fmt?. Indentation looks off.

Reply:
I've done it, thanks for the observation